### ISSUE TITLE
Function reference missing line coverage

### ISSUE THREAD

**Issue by samtwil** [Kotlin/kotlinx-kover#148]
I'm only seeing partial coverage on a function reference when running koverHtmlReport

Here is an example:
```
class Foo {
    fun bar() {
        print("hello")
    }
}

fun main() {
    val foo = Foo()
    print(foo::bar) // <--- Partially covered
}
```

I would expect the `foo::bar` code to be covered since it's just a function reference and not an actual invocation.

Is there someting I'm missing here? Thanks!

gradle version: 7.1
kotlin version: 1.6.0
kotlinx-kover version: 0.5.0

**Comment by Xinkle** [comment]
I got same problem.

**Comment by zuevmaxim** [comment]
Fixed in agent version 1.0.682

**Comment by shanshin** [comment]
Fixed in Kover `0.6.1`
